superset-frontend/src/components/PopoverSection/index.tsx

```diff
@@ -0,0 +1,93 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React, { MouseEventHandler, ReactNode } from 'react';
+import { useTheme } from '@superset-ui/core';
+import { Tooltip } from 'src/common/components/Tooltip';
+import Icon, { IconProps } from 'src/components/Icon';
+
+export interface PopoverSectionProps {
+  title: string;
+  isSelected?: boolean;
+  onSelect?: MouseEventHandler<HTMLDivElement>;
+  info?: string;
+  children?: ReactNode;
+}
+
+export default function PopoverSection({
+  title,
+  isSelected,
+  children,
+  onSelect,
+  info,
+}: PopoverSectionProps) {
+  const theme = useTheme();
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const IconWithoutRef = React.forwardRef((props: IconProps, ref: any) => (
+    <Icon {...props} />
+  ));
+
+  return (
+    <div
+      css={{
+        paddingBottom: theme.gridUnit * 2,
+        opacity: isSelected ? 1 : 0.5,
+      }}
+    >
+      <div
+        role="button"
+        tabIndex={0}
+        onClick={onSelect}
+        css={{
+          display: 'flex',
+          alignItems: 'center',
+          cursor: onSelect ? 'pointer' : 'default',
+        }}
+      >
+        <strong data-test="popover-title">{title}</strong>
+        {info && (
+          <Tooltip title={info} css={{ marginLeft: 5 }}>
```

Comment:
```suggestion
          <Tooltip title={info} css={{ marginLeft: theme.gridUnit }}>
```
I know... I know... I never shut up about these... but since you got the other ones I can't help myself ;)